.github/workflows/ember-assets.yml

```diff
@@ -1,15 +1,11 @@
 name: Ember Asset Sizes
-
 on:
   pull_request:
     paths:
-    - 'ui/**'
-
+      - 'ui/**'
 jobs:
   build:
-
     runs-on: ubuntu-latest
```

Comment:
Is there some kind of yaml linter being used in this PR? There's a lot of fairly arbitrary line and spacing changes going on here.

Reply:
This PR adds `.github/workflows/actionlint.yml` => [rhysd/actionlint](https://github.com/rhysd/actionlint), which does many things, but it doesn't appear to do generic yaml linting, so I wonder if these might be from a person's IDE?